extensions/rest/src/main/java/org/radargun/traits/RESTOperations.java

```diff
@@ -0,0 +1,23 @@
+package org.radargun.traits;
+
+import java.util.List;
+import javax.ws.rs.core.Cookie;
+import javax.ws.rs.core.MultivaluedMap;
+import org.radargun.Operation;
+import org.radargun.WrappedHttpResponse;
+
+/**
+ * @author Martin Gencur
+ */
+@Trait(doc = "Http operations.")
+public interface RESTOperations {
+   String TRAIT = RESTOperations.class.getSimpleName();
+
+   Operation GET = Operation.register(TRAIT + ".Get");
+
+   RESTOperationInvoker getRESTInvoker(String contextPath);
```

Comment:
I've already implemented the Fuzzy logic to evenly spread the load but somehow the changes got lost. Will add it in this PR.